src/visualization/interpolate.jl

```diff
@@ -250,6 +250,100 @@ function unstructured_2d_to_1d(original_nodes, unstructured_data, nvisnodes, sli
   return get_data_1d(reshape(new_nodes[:, 1:new_id], 1, n_nodes_in, new_id), new_unstructured_data[:, 1:new_id, :], nvisnodes)
 end
 
+# Convert 2d unstructured data to 1d data at given coordinates.
+function unstructured_2d_to_1d_along_curve(original_nodes, unstructured_data, nvisnodes, coordinates, mesh, solver, cache)
+
+  @assert size(coordinates) == (2, size(coordinates)[2]) "Coordinates along curve must be 2xn dimensional."
+
+  n_elements_curve = size(coordinates)[2]
+  n_nodes, _, n_elements, n_variables = size(unstructured_data)
+  nodes_in, _ = gauss_lobatto_nodes_weights(n_nodes)
+
+  # Setup data structures.
+  nodes_on_curve = collect(range(0, 1, length = n_elements_curve))
```

Comment:
As stated above: I think here we should actually use the the arc length of the curve instead of just mapping it to [0,1]. If the user wishes so, they can always map it to [0,1] themselves, while the other way around is impossible.

Reply:
Also, in that case this variable could just be named `arc_length`

Reply:
Done